### ISSUE TITLE
SBT console arrows up or down don't show history

### ISSUE THREAD

**Issue by gilcu2** [orfjackal/idea-sbt-plugin#116]
In Android Studio 1.5.1 sbt console move cursor up or down instead of move in history when press up or down arrows.

**Comment by doncorsean** [comment]
yeah, please fix this. This was working at some point. Is there a workaround?
